open-apparel-registry/open-apparel-registry
# Minor UX changes

## Overview

- Prevents double loading spinners in the search sidebar
- Updates font sizes and copy in the search sidebar 
- Moves 'search by area' to the map
- Moves 'copy search' to the sidebar
- Removes the 'Guide' tab

Connects #1743 

## Demo

<img width="1258" alt="Screen Shot 2022-03-20 at 4 06 52 PM" src="https://user-images.githubusercontent.com/21046714/159183979-e9446048-aab2-488a-8364-21afa8f11467.png">

<img width="321" alt="Screen Shot 2022-03-20 at 4 07 07 PM" src="https://user-images.githubusercontent.com/21046714/159183985-e90bf807-65ce-4106-a114-e76f3f45b5bf.png">

<img width="323" alt="Screen Shot 2022-03-20 at 4 07 50 PM" src="https://user-images.githubusercontent.com/21046714/159184007-16f9ce58-67ef-4eba-a104-8a5661709787.png">

## Testing Instructions

* On develop, run `./scripts/server`. Search for a 'number of workers' in the sidebar. Refresh the page to see double loading spinners (on small screens, you might need to zoom out slightly so the search button and a bit of space above it show by default). 
* On this branch, run `./scripts/server`. Search for a 'number of workers' in the sidebar. Refresh the page and you should only see one spinner. 
* Search by area using the control on the map. 
* Copy the search address using the 'copy link' button. 
* Confirm that the tooltip text and the explanatory text (in the nested extended fields section) match. 
* Confirm the search sidebar matches [the wireframes](https://www.figma.com/file/x4ae8NNWyGfSo7spUvH5u1/Open-Apparel-Registry?node-id=53%3A565). 

## Checklist
